.circleci/config.yml

```diff
@@ -87,6 +87,21 @@ jobs:
             export SONAR_API_TOKEN=$SONAR_TOKEN
             ./.ci/validation.sh sonarqube
 
+  yamllint:
+    docker:
+      - image: cimg/base:2022.11
+
+    steps:
+      - checkout
+      - run:
+          name: Install dependencies
+          command: |
+            sudo apt update
+            sudo apt install -y yamllint
```

Comment:
why dont make a specific image with yamllint installed instead of running these commands for every run?

Reply:
The task completes in 14 seconds on the base cimg (should be in the cache on every machine). I would wager that creating a custom image would cause this to actually take longer due to the fact that it is likely to not be cached in many cases.

Also, saving 2-3 seconds is not worth the cost to maintain this image. I would consider this if the package was not in apt for stability's sake, but it is overkill in this case.

Reply:
Yes, better to avoid custom images. Download custom image might be slower, as CI pre cache a lot of stuff to use standard images quickly.

Several years ago situation was a bit different.